test_scripts/Defects/7_1/1619_AlertManeuer_empty_SoftButtons_missing_Info.lua

```diff
@@ -0,0 +1,75 @@
+--------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1619
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with valid AlertManeuver response (empty parameter "info")
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) App is registered
+-- 4) PTU
+-- 5) App is activated
+--
+-- Steps:
+-- 1) Send AlertManeuver mobile RPC from app with parameter softButtons = []
+-- 2) HMI responds with SUCCESS to Navigation.AlertManeuver request
+--
+-- Expected:
+-- 1) App receives AlertManeuver response, with { success = true, resultCode = "SUCCESS" }
+--    and missing parameter "info"
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local json = require('modules/json')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local params = {
+  softButtons = json.EMPTY_ARRAY
+}
+
+--[[ Local Functions ]]
+local function sendAlertManeuver()
+  local mobileSession = common.mobile.getSession()
+  local hmi = common.hmi.getConnection()
+  local dataToHMI = {
+    appID = common.app.getHMIId()
+  }
+
+  local cid = mobileSession:SendRPC("AlertManeuver", params)
+  hmi:ExpectRequest("Navigation.AlertManeuver", dataToHMI)
+  :Do(function(_, data)
+      hmi:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+
+  mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  :ValidIf(function(_,data)
+    if data.payload.info == nil then
```

Comment:
`SDL responds with not valid info param` not correct message because this info is checked on the mobile side and validation checks the only presence of info parameter but not validness
Propose to update if like
  if data.payload.info then
    return false, "Mobile response contains not expected info param"
  end
  return true